src/webview/components/memory-table.tsx

```diff
@@ -93,13 +94,9 @@ export const MoreMemorySelect: React.FC<MoreMemorySelectProps> = ({ count, offse
     );
 };
 
-interface MemoryTableProps extends TableRenderOptions, MemoryDisplayConfiguration {
-    memory?: Memory;
-    decorations: Decoration[];
-    offset: number;
-    count: number;
-    fetchMemory(partialOptions?: Partial<DebugProtocol.ReadMemoryArguments>): Promise<void>;
-    isMemoryFetching: boolean;
+interface MemoryTableProps {
+    endianness: Endianness;
```

Comment:
Unrelated to your changes, but it seems we don't really ever use the `endianness` values we pass around.

Reply:
Just noticed the same. We have "Selectable word endianess" in #48 . But I was falsely assuming before that the word display was little endian already which seems to be the internal default. I've raised https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/issues/88